src/Silverback.Core/Diagnostics/EventIds.cs

```diff
@@ -0,0 +1,157 @@
+﻿// Copyright (c) 2020 Sergio Aquilini
+// This code is licensed under MIT license (see LICENSE file for details)
+
+using Microsoft.Extensions.Logging;
+
+namespace Silverback.Diagnostics
+{
+    /// <summary>
+    ///     This class contains all EventIds used for log entries.
+    /// </summary>
+    public static class EventIds

```

Comment:
I personally would have created a sort of structure in here, nesting some static classes.

```
static class EventIds
{
  static class SkipMessagePolicy
  {
    public static EventId MessageSkipped { get; } = new EventId(...)
  }
}
```

But that's just me. :)

Reply:
I'm fine to do some more adjustments. See my comment below.